packages/tradershub/src/features/cfd/flows/CTrader/AddedCTraderAccountsList/AddedCTraderAccountsList.tsx

```diff
@@ -18,10 +18,12 @@ const LeadingIcon = () => (
 
 const AddedCTraderAccountsList = () => {
     const { data: cTraderAccounts } = useCtraderAccountsList();
-    const { data: activeTrading } = useActiveTradingAccount();
+    const { data: activeTradingAccount } = useActiveTradingAccount();
```

Comment:
@mitra-deriv per our your own suggestion let's mention `deriv` explicitly
```suggestion
    const { data: activeDerivTrading } = useActiveTradingAccount();
```

Reply:
I believe we are using this in many places I used what we using now, maybe better I open a PR and address it in all files?

Reply:
how about `activeDerivTradingAccount`

Reply:
if it is being used in many places then better to keep it as it is. It is just a minor improvement.